backend/alembic/versions/ff2ed634a1e9_initial_vending_models.py

```diff
@@ -0,0 +1,36 @@
+"""Initial vending models
+
+Revision ID: ff2ed634a1e9
+Revises: 7589e5e508b9
+Create Date: 2022-03-30 12:58:55.002060
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ff2ed634a1e9'
+down_revision = '7589e5e508b9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('stripeexpressaccount',
```

Comment:
Hrm, having this non generic might adding an additional provider like paypal problematic. Still, we could just migrate the database then, so shouldn't be a blocker.